libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/list/InstructionListAdapter.java

```diff
@@ -70,7 +68,11 @@ public void onViewDetachedFromWindow(InstructionViewHolder holder) {
     holder.itemView.clearAnimation();
   }
 
-  public void updateSteps(RouteProgress routeProgress) {
+  public void updateSteps(Context context, RouteProgress routeProgress,
+                          Locale locale, @NavigationUnitType.UnitType int unitType) {
+    if (distanceUtils == null) {
```

Comment:
Would it make sense to store `locale` or `unitType` and then check if they have changed / re-initialize `DistanceUtils` here if they have?  I believe the formatting elsewhere (like in `InstructionView`) can change on the fly.

Reply:
@danesfeder I agree, we should probably add something like that to `InstructionStepResource` too so we're not creating a new `DistanceUtils` object everytime